_posts/2019-11-04-unity-rotate-around-vs-physics-based-approach.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: Unity Space Simulator - Rotate Around VS Physics Based Approach
+date: 2019-11-04 16:28:05.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [Programming]
+image:
+image2:
+author: Valentino Urbano
+---
+
+[This is part 2 of my Unity Space Series. You should first read part 1 if you haven't already][1]
+
```

Comment:
No "."?

Reply:
I think it sounds better with the pause. Maybe a comma?

Reply:
Not what I meant...

Reply:
Makes more sense =)